suse_caasp

```diff
@@ -94,6 +94,39 @@ conf_files() {
         done
 }
 
+# Input: logfilename lines logfiles...
+# If lines = 0, includes the entire log file
+log_files() {
```

Comment:
I'm a bit confused. this function does not seem to be used? did I miss something?

Reply:
It's called from `find_and_log_files`, implemented in this commit:
https://github.com/SUSE/supportutils-plugin-suse-caasp/commit/df6791c11f2ce5331ab27290be82d5b823245d0b

Weird. :)

Reply:
That's right `log_files()` function is called in `find_and_log_files` and it was accidentally removed. We have to bring it back

Reply:
ah, so it was not added as part of this comment. I was only looking at this PR, thus I did not see it's usage. I figured because it was a new function it's use would have to be in this PR :) cool.